docs/configuration-v1_1-exp.md

```diff
@@ -56,6 +56,7 @@ The Fedora CoreOS configuration is a YAML document conforming to the following s
     * **_label_** (string): the label of the filesystem.
     * **_uuid_** (string): the uuid of the filesystem.
     * **_options_** (list of strings): any additional options to be passed to the format-specific mkfs utility.
+    * **_with_mount_unit_** (bool): Whether to generate a generic mount unit for this filesystem as well. If a more specific unit is needed, a custom one can be specified in the `systemd.units` section. The unit will be named with the [escaped][systemd-escape] version of the `path`.
```

Comment:
s/bool/boolean/ (for consistency with other entries).
Also, this would benefit from an additional "Defaults to false" sentence.